src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/src/extension.ts

```diff
@@ -20,7 +20,7 @@ export async function activate(context: vscode.ExtensionContext) {
     const config = process.env.config ? process.env.config : 'Debug';
 
     const languageServerDir = path.join(
-        __dirname, '..', '..', '..', '..', '..', 'artifacts', 'bin', 'rzls', config, 'net6.0');
+        __dirname, '..', '..', '..', '..', '..', 'artifacts', 'bin', 'rzls', config, 'net7.0');
```

Comment:
Since the last time we inserted, our langauge server directory changed from net6.0 -> net7.0